src/Questionnaire/Questionnaire.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from 'react'
+import { questionnaires } from '../data/questionnaires'
+
+export default function Questionnaire() {
+  const { questions } = questionnaires[0]
```

Comment:
You are right, I shouldn't have saved the datait in an array for the first story. I wanted to show the number of the questionnaire to the user but later decided against it. 
I will need this for the second story, so I think it will be extra-work to remove it now and then add it again, when I start with the second story. But for the next features I'll try to only build what I need for the current story.